client/app/src/main/java/group01/smartcar/client/CarControl.java

```diff
@@ -19,26 +18,32 @@
     private final String SUBSCRIBE_URI = "/smartcar/control/#";
     private final String STEERING_URI = "/smartcar/control/steering";
     private final String THROTTLE_URI = "/smartcar/control/speed";
+    private final String CAMERA_URI = "/smartcar/control/camera";
```

Comment:
Not an issue, just an idea. Maybe we can separate information in the MQTT sub-channels. Maybe the "control" channel is not the best place to publish and receive camera messages. We can use a new sub-channel for sensors. Just an opinion.

Reply:
Do you have a suggestion for a different channel?

Reply:
"/smartcar/camera" or "/smartcar/sensors/camera" can be alternatives.